packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -166,9 +167,9 @@ export default class SelectOpener extends React.Component<
                 aria-label={ariaLabel ?? undefined}
                 aria-required={ariaRequired}
                 aria-haspopup="listbox"
+                aria-controls={ariaControls}
```

Comment:
The `role=combobox` requires `aria-controls` to be set. The `aria-controls` attribute was already being set by the parent component and it was applied to the button via the `...sharedProps`. We now explicitly set it to address the error!